apps/condo/domains/common/constants/stopWords.js

```diff
@@ -0,0 +1,40 @@
+const ADDRESS_SEARCH_STOP_WORDS_RU = [
+    'д',
+    'дом',
+    'к',
+    'корпус',
+    'зд',
+    'здание',
+    'стр',
+    'строение',
+    'ул',
+    'улица',
+    'пр-кт',
+    'проспект',
+    'пер',
+    'переулок',
+    'г',
+    'город',
+    'дер',
+    'деревня',
+    'пос',
+    'поселок',
+    'с',
+    'село',
+    'тер',
+    'территория',
+    'респ',
+    'республика',
+    'р-н',
+    'р-он',
+    'район',
+]
+
+// NOTE: If you really need, you can add stop words for other languages
+const ADDRESS_SEARCH_STOP_WORDS = [
```

Comment:
How about moving this in aggregate `index` file?

Reply:
is it really worth splitting the contents of this file?

Reply:
For now it's optional and ok. But it can become unreadable on scale